NewForce-Cohort-9/tabloid-full-stack-full-stack-force
# Fw ticket38 create reaction

Create Reactions
Description
Added Create Reactions function in Post Details to Create reactions. [Trello Ticket #38 ](https://trello.com/c/yoHhckhW/38-38-create-new-reaction)
Type of Change
-Added new, non-breaking feature.
Testing Instructions
1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout fw-ticket38-addReaction
5. If you don't see the most recent version of my files, git pull origin fw-ticket38-addReaction
6. Inside of Visual Studio (purple), click the execute with debugger solid green button to launch the app
7. In your terminal, navigate to where your package.json file is in your client folder. Run "npm start" in your terminal.
8. Login as an admin. foo@bar.com for the email parameter. Leave the password text box blank. Click "Login".
9. Click on Posts at the top of the navbar.
10. Once on the posts page, click on any post. In the post details, you should see a button Create New Reaction. 
11. Click on the button to be led to a form to create a new reaction. Input your new reaction name and url then click Add Reaction button. 
12. You will be led back to post details. Check your new reaction with count appears on post details.
Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
